src/main/java/lotto/domain/Ball.java

```diff
@@ -0,0 +1,53 @@
+package lotto.domain;
+
+import lotto.system.ExceptionMessage;
+import lotto.system.LottoNumberConstant;
+import lotto.system.SystemMessage;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+public class Ball {
+    private final int number;
+
+    private Ball(int number) {
+        validate(number);
+        this.number = number;
+    }
+
+    public static Ball from(int number) {
+        return new Ball(number);
+    }
+
+    public static Ball createManual() {
+        while (true) {
+            OutputView.printFrom(SystemMessage.INPUT_BONUS_NUMBER);
+            try {
+                return new Ball(InputView.readInt());
+            } catch (IllegalArgumentException e) {
+                OutputView.exceptionMessage(e);
+            }
+        }
+    }
+
+    private static void validateRange(int number) {
+        if (isUnderRange(number) || isOverRange(number)) {
+            throw new IllegalArgumentException(ExceptionMessage.LOTTO_RANGE.getMessage());
+        }
+    }
+
+    private static boolean isUnderRange(int number) {
+        return number < LottoNumberConstant.MIN.getValue();
+    }
+
+    private static boolean isOverRange(int number) {
+        return number > LottoNumberConstant.MAX.getValue();
+    }
```

Comment:
validateRange의 가독성을 위해서 분리해주신 것 같아요! 왠지모르게 제가 이전 주차에 남겨드린 리뷰를 적용해주신 것 같아서 감동입니다 😭 (아니라면 저의 망상으로 끝나는 것으로 하겠습니다 ㅋㅋ;;)

다만 제 생각엔 두개의 메서드를 합쳐서 isInvalidRange 메서드로 만들었어도 좋을 것 같습니다!
```suggestion
    private static boolean isInvalidRange(int number) {
        //.....
    }
```

Reply:
2주차 리뷰 남겨주신 것 반영한거 맞습니다 🥹 기억해주셨네요 👍👍
확실히 말씀하신대로 너무 과하게 파편화한 것 같아요 🥲

Reply:
저도 보면서 굳이 안나눠도 될 것 같다고 생각이 드네요